torch/onnx/__init__.py

```diff
@@ -304,6 +304,17 @@ def forward(self, x):
             particular types of modules to export as local functions in ONNX.
             This feature requires ``opset_version`` >= 15, otherwise the export will fail. This is because
             ``opset_version`` < 15 implies IR version < 8, which means no local function support.
+            Module variables will be exported as function attributes. There are two categories of function
+            attributes.
+
+            1. Annotated attributes: class variables that are annotated will be exported as attributes.
```

Comment:
```suggestion
            1. Annotated attributes: class variables that have type annotations be exported as attributes.
```

Reply:
thx, I'm adding the link of PEP 526 to provide extra clarity.

            1. Annotated attributes: class variables that have type annotations via
            `PEP 526-style <https://www.python.org/dev/peps/pep-0526/#class-and-instance-variable-annotations>`_
            will be exported as attributes.